Tests/SUUpdateValidatorTest.swift

```diff
@@ -62,14 +62,16 @@ class SUUpdateValidatorTest: XCTestCase {
         let dsaSig: String?
         switch config.dsa {
         case .none: dsaSig = nil
-        case .invalid: dsaSig = "MCwCFCIHCiYYkfZavNzTitTW5tlRp/k5AhQ40poFytqcVhIYdCxQznaXeJPJDQ=="
+        // Use some invalid base64 characters
+        case .invalid: dsaSig = "^%wCFCIHCIYYkfZavNzTitTW5tlRp/k5AhQ40poFytqcVhIYdCxQznaXeJPJDQ=="
```

Comment:
This wasn't actually generating an invalid signature status before.

Reply:
Never mind. Invalid here meant still formatted in valid way, but bad in verification. I added another configuration case for invalidFormat.